### PR TITLE
Add chinese version

### PR DESCRIPTION
**Pull request by Aisuko**
Signed-off-by: aisuko <urakiny@gmail.com>

**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Aisuko** [comment]
Here @Micost

**Review by gyohuangxin — COMMENTED** [review]
@Aisuko Great, and I leave suggestion for some typo.

**Comment by Aisuko** [comment]
Hi @warunicorn19 , Would you mind help us to check the CI error. I saw you are the master on this project.

**Comment by warunicorn19** [comment]
Sure @Aisuko, i will take a look 😃

File: src/collections/events/kubecon-china-2021/index.mdx [hunk 1]
```diff
@@ -28,4 +28,76 @@ Speakers:
...
+
+<h2>Meshery project office hours</h2>
+
+<p>Join the Meshery project office hours at KubeCon China 2021 to learn more about the CNCF's latest service mesh project and its maintainers.</p>
+<p>Come and discover why Meshery is the easiest way to get started with 10+ service meshes!</p>
+
+
+Speakers:
+<ul>
+<li><Link to="/community/members/aisuko-li">Aisuko Lee</li>
```
Comment (warunicorn19) [inline src/collections/events/kubecon-china-2021/index.mdx:66]:
```suggestion
<li><Link to="/community/members/aisuko-li">Aisuko Lee</Link></li>
```

**Review by warunicorn19 — CHANGES_REQUESTED** [review]

File: src/collections/events/kubecon-china-2021/index.mdx [hunk 2]
```diff
@@ -28,4 +28,76 @@ Speakers:
...
+
+<h2>Meshery project office hours</h2>
+
+<p>欢迎大家加入到KubeCon 2021 中国站 Meshery office hours，来了解CNCF服务网格项目的最新进展，以及与项目维护者进行在线互动。 </p>
+<p>快来了解为什么Meshery是开始使用10+个服务网格的最简单方法！</p>
+
+
+演讲者:
+<ul>
+<li><Link to="/community/members/aisuko-li">Aisuko Lee</li>
```
Comment (warunicorn19) [inline src/collections/events/kubecon-china-2021/index.mdx:90]:
```suggestion
<li><Link to="/community/members/aisuko-li">Aisuko Lee</Link></li>
```

**Review by warunicorn19 — CHANGES_REQUESTED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit 93f72812978e3b8019308950876e7c38dd5cebbd at: https://61a0e6e760728c0dc2a57e51--layer5ng.netlify.app

**Review by warunicorn19 — APPROVED** [review]
@Aisuko fixed the build fail. Merge at will.

**Comment by navendu-pottekkat** [comment]
@Aisuko Could we go English, Mandarin, English, Mandarin ...

Or English first completely and then Mandarin?

I think either of these would be better.

Let me also confirm the other participants.

**Comment by Aisuko** [comment]
> @Aisuko fixed the build fail. Merge at will.

Thank you @warunicorn19 👍

**Comment by Aisuko** [comment]
> @Aisuko Could we go English, Mandarin, English, Mandarin ...
> 
> Or English first completely and then Mandarin?
> 
> I think either of these would be better.
> 
> Let me also confirm the other participants.

Yes, the format that you said is very clear than current. English go first completely and than Mandarin would be good.

**Comment by l5io** [comment]
🚀 Preview for commit 28f7f3c39b17a04343e036cb0fade6c21b076966 at: https://61a2db09f91bffb8d106d788--layer5ng.netlify.app

**Review by leecalcote — APPROVED** [review]
Thank you, gents. This is excellent.

**Comment by leecalcote** [comment]
@gyohuangxin, if we can get you started on running service mesh performance benchmarks, then let's get your name on the SMP office hours and you into the discussion (office hours) at KubeCon.

**Comment by gyohuangxin** [comment]
> @gyohuangxin, if we can get you started on running service mesh performance benchmarks, then let's get your name on the SMP office hours and you into the discussion (office hours) at KubeCon.

@leecalcote Yes, I can run SMP and get started on developing SMP Github action, it's good for me to join the office hours, thanks! Please let me know if I can be of any help.

**Pull request merged**